### PR TITLE
23521 - update continuation emails

### PR DESCRIPTION
**Pull request by ketaki-deodhar**
*Issue #:* /bcgov/entity#23521

*Description of changes:*

- updated Continuation Authorization emails
- updated Continuation completion emails

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

**Review by ArwenQin — APPROVED** [review]
LGTM!

File: queue_services/entity-emailer/src/entity_emailer/email_templates/NR-TEST.html
Comment (vysakh-menon-aot) [inline queue_services/entity-emailer/src/entity_emailer/email_templates/NR-TEST.html:1]: Why do we have this file?

**Comment by vysakh-menon-aot** [comment]
As Mihai mentioned in todays standup we need to feature flag the NR template changes (you can create separate PR for NR templates with FF)

File: queue_services/entity-emailer/src/entity_emailer/email_templates/NR-TEST.html
Comment (ketaki-deodhar) [reply queue_services/entity-emailer/src/entity_emailer/email_templates/NR-TEST.html:1]: missed removing it. Done now

**Comment by ketaki-deodhar** [comment]
> As Mihai mentioned in todays standup we need to feature flag the NR template changes (you can create separate PR for NR templates with FF)

Sure. Will do. I believe there will be discussion about it and will apply changes after that.

**Comment by vysakh-menon-aot** [comment]
> > As Mihai mentioned in todays standup we need to feature flag the NR template changes (you can create separate PR for NR templates with FF)
> 
> Sure. Will do. I believe there will be discussion about it and will apply changes after that.

You have to exclude NR templates from this PR (since it should not go to PROD without FF)

**Comment by ketaki-deodhar** [comment]
> > > As Mihai mentioned in todays standup we need to feature flag the NR template changes (you can create separate PR for NR templates with FF)
> > 
> > 
> > Sure. Will do. I believe there will be discussion about it and will apply changes after that.
> 
> You have to exclude NR templates from this PR (since it should not go to PROD without FF)

You mean take NR templates from this PR altogether?

**Comment by ketaki-deodhar** [comment]
Will revert all changes related to NR emails

**Review by vysakh-menon-aot — APPROVED** [review]

**Pull request merged**